.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Just like with the .idea, we likely don't want vscode settings to be included in the respository, but maybe I'm wrong.